### PR TITLE
Add `rgb_to_hsv` and `hsv_to_rgb`

### PR DESCRIPTION
**Pull request by james77777778**
These two functions are useful for image prerpocessing and augmentation

Most of the implementations are inspired by:
https://github.com/google-deepmind/dm_pix

EDITED:
Decrease the values of `atol` in `keras/src/ops/image_test.py`. They were set too high.

### CODE REVIEW

File: keras/src/ops/image.py [hunk 1]
```diff
@@ -76,6 +76,147 @@ def rgb_to_grayscale(images, data_format=None):
...
+            If not specified, the value will be interpreted by
+            `keras.config.image_data_format`. Defaults to `None`.
+
+    Returns:
+        HSV image or batch of HSV images.
+
+    Examples:
+
+    >>> import numpy as np
+    >>> from keras.src import ops
```
Comment (fchollet) [inline keras/src/ops/image.py:130]: `from keras import ops`

File: keras/src/ops/image.py [hunk 2]
```diff
@@ -76,6 +76,147 @@ def rgb_to_grayscale(images, data_format=None):
...
+            If not specified, the value will be interpreted by
+            `keras.config.image_data_format`. Defaults to `None`.
+
+    Returns:
+        RGB image or batch of RGB images.
+
+    Examples:
+
+    >>> import numpy as np
+    >>> from keras.src import ops
```
Comment (fchollet) [inline keras/src/ops/image.py:199]: Same here

File: keras/src/ops/image.py [hunk 3]
```diff
@@ -76,6 +76,147 @@ def rgb_to_grayscale(images, data_format=None):
...
+
+    Args:
+        images: Input image or batch of images. Must be 3D or 4D.
+        data_format: A string specifying the data format of the input tensor.
+            It can be either `"channels_last"` or `"channels_first"`.
+            `"channels_last"` corresponds to inputs with shape
+            `(batch, height, width, channels)`, while `"channels_first"`
+            corresponds to inputs with shape `(batch, channels, height, width)`.
+            If not specified, the value will be interpreted by
+            `keras.config.image_data_format`. Defaults to `None`.
```
Comment (fchollet) [inline keras/src/ops/image.py:191]: "the value will default to `keras.config.image_data_format()`"

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: keras/src/ops/image.py (hunk 1 above)
Comment (james77777778) [reply keras/src/ops/image.py]: Fixed

File: keras/src/ops/image.py (hunk 2 above)
Comment (james77777778) [reply keras/src/ops/image.py]: Fixed

File: keras/src/ops/image.py (hunk 3 above)
Comment (james77777778) [reply keras/src/ops/image.py]: Fixed and applied to all other image ops.

**Review by fchollet — APPROVED** [review]
LGTM, thank you!

**Pull request merged**